src/params.js

```diff
@@ -31,15 +31,16 @@ export function useParamsStore() {
                 return s.state.user.LanguageLocaleKey.split('_').join('-')
             }
         }),
-
         language: computed(() => {
             if (s.state.salesForceUser) {
                 return s.state.salesForceUser.LanguageLocaleKey.split('_')[0].toLowerCase()
             }
             if (s.state.user) {
                 return s.state.user.LanguageLocaleKey.split('_')[0].toLowerCase()
             }
-            if (s.state.config) {
```

Comment:
could we return in language the deviceLanguage if the sfdc language is not available?